sources/zafira-web/angular/client/app/_testruns/testrun.controller.js

```diff
@@ -1414,18 +1417,12 @@
             return count;
         };
 
-        $scope.splitPlatform = function (string) {
-	    if (string == null) {
-		return null;
-	    }
-            var array = string.split(' ');
-            var version = "v." + array[1];
-            if (array.length == 2) {
-                return version;
-            }
-            else {
-                return null;
+        var splitPlatform = function (string) {
+            var version = null;
+            if (string && string.split(' ').length > 1) {
+                version = "v." + string.split(' ')[1]
```

Comment:
Why do we need “v.”?

Reply:
just for display near platform icon

Reply:
![screen shot 2018-08-14 at 11 38 24 am](https://user-images.githubusercontent.com/23284071/44081094-991926b4-9fb6-11e8-86e9-ee8a8b9b3648.png)

Reply:
ah, that's js... all good